VoltAir/qml/ui/LoadingScreen.qml

```diff
@@ -108,6 +108,6 @@ UiComponent {
         width: loading.width
         height: 0.25 * loading.height
 
-        textElement.text: "Loading"
+        textElement.text: TR.value("Loading")
```

Comment:
@diegowald How does this work? Does the `TR` utility look up a text stored in a text file with the property value "Loading"?